Makefile

```diff
@@ -108,7 +122,7 @@ ifeq (, $(shell which kustomize))
 	KUSTOMIZE_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$KUSTOMIZE_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/kustomize/kustomize/v3@v3.5.4 ;\
+	go get sigs.k8s.io/kustomize/kustomize/v4@v4.0.5 ;\
```

Comment:
Have you tested the `make before-pr` with this new kustomize version? I'm assuming it's working, but they are known to break things between releases. :(

Reply:
yeah ... did this change because it is the one used in GHAction PR check ... will run the `before-pr` to be sure